chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -140,6 +140,16 @@ public class Scene implements JsonSerializable, Refreshable {
    */
   public static final double DEFAULT_FOG_DENSITY = 0.0;
 
+  /**
+   * Minimum branch count.
+   */
+  public static final int MIN_BRANCH_COUNT = 1;
+
+  /**
+   * Maximum branch count.
+   */
+  public static final int MAX_BRANCH_COUNT = 50;
```

Comment:
These are pretty much GUI-only constants, should we move them into the advanced tab?

Reply:
Seems reasonable, will do.